Packs/Base/Scripts/DBotSuggestClassifierMapping/DBotSuggestClassifierMapping.py

```diff
@@ -526,8 +533,25 @@ def get_alias_index(field_name, alias):
     return SIEM_FIELDS[field_name]['aliases'].index(alias)  # type: ignore
 
 
-def get_most_relevant(field_name, field_mappings):
-    candidates = sorted(field_mappings, key=lambda x: get_alias_index(field_name, x[1]))
+def get_most_relevant_json_field(field_name, json_field_to_alias):
+    if not json_field_to_alias:
+        return
+
+    # calculate jaccard score for each alias, and get the candidates with max score
+    scores = {}
+    for json_field, alias in json_field_to_alias.items():
+        scores[json_field] = jaccard_similarity_for_string_terms(json_field, alias)
+    scores = {k: v for k, v in scores.items() if v == max(scores.values())}
+
+    # calculate jaccard score for each field, and get the candidates with max score
+    for json_field, alias in json_field_to_alias.items():
+        scores[json_field] = jaccard_similarity_for_string_terms(json_field, field_name)
+    scores = {k: v for k, v in scores.items() if v == max(scores.values())}
```

Comment:
In case ` jaccard_similarity_for_string_terms(json_field, alias)` is high, it's possible that you'll return a mapping between json_field to its alias?

Reply:
json field doesn't have alias inciden field it's the one with alias
so the match is json field with incident field alias